app/serializers/form1010cg/submission_serializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Form1010cg
+  class SubmissionSerializer < ActiveModel::Serializer
+    attribute(:confirmation_number) { object.carma_case_id }
+    attribute(:submitted_at) { object.submitted_at }
```

Comment:
Turns out `attribute :submitted_at` will not work since the model is not an ActiveRecord instance. So I will need to use the more verbose version for now.

Reply:
huh, interesting.  I guess they're serious about the `active_model` part of `active_model_serializers` 😄